apps/server/src/schemas/apiKey.schema.ts

```diff
@@ -0,0 +1,16 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+import { Recipient, RecipientSchema } from './recipient.schema';
+
+export type ApiKeyDocument = HydratedDocument<ApiKey>;
+
+@Schema()
+export class ApiKey {
+  @Prop()
+  key: string;
+
+  @Prop()
+  name: string;
+}
+
+export const ApiKeySchema = SchemaFactory.createForClass(ApiKey);
```

Comment:
Small thing. If you can add a empty line at the end of each file.  
To prevent this problem further, we could configure this stuffs for the whole project with eslint and prettier file.

Reply:
Added to all files, I can pick the prettier and eslint configuration to the project.